docs/development/patterns.rst

```diff
@@ -81,7 +81,7 @@ process for adding new dependencies:
 1. Remove the project name from the appropriate ``.in`` file
 2. From the repositories root directory, recompile the dependencies for each modified ``.in`` file::
```

Comment:
I think this line needs updated too

Reply:
Hm ? I'm not sure I understand how it should be replaced?

The best I understand from this is that you imply we should talk about the `.txt` file ? Like:
```suggestion
2. From the repositories root directory, recompile the ```.txt` for each modified ``.in`` file::
```

Reply:
Sorry, read this too fast, you're right